packages/sitecore-jss-nextjs/src/middleware/redirects-middleware.test.ts

```diff
@@ -857,7 +863,8 @@ describe('RedirectsMiddleware', () => {
           url: 'http://localhost:3000/found',
           setCookies,
         });
-        const nextRewriteStub = sinon.stub(NextResponse, 'rewrite').callsFake((url) => {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        const nextRewriteStub = sinon.stub(NextResponse, 'rewrite').callsFake((url, _init) => {
```

Comment:
similarly here, should _init be used?

Reply:
@art-alexeyenko Same as above, we are calling the two parameter version of this method, so matching the call signature for the fake.